.env.example

```
@@ -37,16 +38,35 @@ RUN_MIGRATION=true
+# IDP_PUBLIC_URL is ThunderID's own public URL (thunderid/deployment.yaml hardcodes
+# the same value as server.public_url). Must be https:// to match deployment.yaml —
+# thunderid-setup's TLS listener rejects plain http. compose.yml feeds it to PDP/CE
+# as IDP_ISSUER directly (there's no separate IDP_ISSUER var to set here), since
+# ThunderID issues tokens with iss == its public URL.
+#
+# consent-engine's verifier validates issuer, audience, and client_id when each is
+# set, extracts the email claim, and is otherwise IdP-agnostic.
+IDP_PUBLIC_URL=https://localhost:8090
+# The OE resource server identifier declared in thunderid/bootstrap/resource.yaml —
+# tokens requested with resource=http://api.openndx.local carry this as `aud`.
+IDP_AUDIENCE=http://api.openndx.local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDP_AUDIENCE key should go before the IDP_PUBLIC_URL key

(UnorderedKey)